libcanard/canard.h

```diff
@@ -306,6 +306,13 @@ struct CanardMemoryResource
     CanardMemoryAllocate   allocate;        ///< Shall be a valid pointer.
 };
 
+/// Holds the statistics of a transmission queue.
+struct CanardTxQueueStats
+{
+    /// Holds number of dropped TX frames (due to timeout when `now > deadline`).
```

Comment:
It's not just timeout. If the media fails to accept a frame, we discard it, right? Along with every other frame in the transfer chain.

Reply:
Such business logic was out of lizard scope. Yes, at libcyphal I did it as you described. At flux_grip and telega I'm not 100% sure, but it seems that there is no notion of error there - just boolean either accepted or not accepted `ifc->push`.

Reply:
Sure, but if we are designing a template method ("template" in the sense of design patterns, not type-generic programming), then it makes sense to support fallible media, right?